### ISSUE TITLE
Thumbnail Issues

### ISSUE THREAD

**Issue by ntantum** [CollegeOfTheHolyCross/dta_sufia#126]
Hi Steven -  

We are having some issues with thumbnails of items again. This seems like the same issue as last week because all the metadata is saving, but the thumbnails are not working. When we tried to put the file into the object again, we just get an error message and are no longer able to edit it. Is there an equivalent to the "refresh thumbnail" button that we used to have? Clearly, my solution of re-adding the file is not working!

Here are the problem objects so far:
[4f16c289p](https://www.digitaltransgenderarchive.net/files/4f16c289p)
[m039k512x](https://www.digitaltransgenderarchive.net/files/m039k512x)
[k643b143](https://www.digitaltransgenderarchive.net/files/k643b143)
[fb494841c](https://www.digitaltransgenderarchive.net/files/fb494841c)

Let me know if you end up needing more information or need the actual files for these items.

Thanks!

**Comment by ntantum** [comment]
Here are two items that I left alone. Instead of trying to re-upload the items fix the thumbnails, I left it so that you could see what the item looked like before I did that. So these items are currently fine except for the thumbnail.

[h702q636h](https://www.digitaltransgenderarchive.net/files/h702q636h)
[f4752g73w](https://www.digitaltransgenderarchive.net/files/f4752g73w)

**Comment by scande3** [comment]
@ntantum -> These need to have their file reuploaded or to be deleted. I'll resolve this with a more permanent fix tomorrow that should give me more debug information on why this might occur. In theory, it should only occur with an error case that I don't see any information about an error message showing with these? 

Thanks!

-Steven

**Comment by ntantum** [comment]
Hi Steven - 
It seems like the error messages went away so I am able to view the objects, however I was not able to just reupload the files of the items. I had to delete them all and re-upload them as new objects.

Just so you can see the error messages:

When I tried to change the file (in hopes of regenerating the thumnail), I got this error message: 
<img width="1400" alt="raylan error 1" src="https://user-images.githubusercontent.com/30127831/38032280-3e0df316-326b-11e8-846f-b214e14281b8.png">
(this identical error message appeared across all items with this issue)

and then this error message when I tried to open the file from the item description page:
<img width="1093" alt="raylan error 2" src="https://user-images.githubusercontent.com/30127831/38032281-3e34d1fc-326b-11e8-8230-cc0612586783.png">
(also identical across items)


**One last issue:**
I am still having an issue with fb494841c. I deleted that object and re-uploaded it as a new object so you can now find it at [7w62f822v](https://www.digitaltransgenderarchive.net/files/7w62f822v). As you can see, this item (along with [h702q636h](https://www.digitaltransgenderarchive.net/files/h702q636h) and [f4752g73w](https://www.digitaltransgenderarchive.net/files/f4752g73w)) is still having a thumbnail issue. Everything else is fine (there is no error message) but I cannot manage to figure out how to fix the thumbnails. Let me know if you have any further insight as to how I can fix this! 

Thanks!

**Comment by ntantum** [comment]
This just happened again with object [9880vq98q](https://www.digitaltransgenderarchive.net/files/9880vq98q). It seems to be only plaguing the Discovery Resources Collection for some reason.

**Comment by scande3** [comment]
@ntantum : Can you attach or email me the file for the ones that you are having difficulty with? Apologies for this and I'll also fix the edit upload on existing objects that this occurs in.

**Comment by ntantum** [comment]
The four items that we are having issues with are:

Item: [9880vq98q](https://www.digitaltransgenderarchive.net/files/9880vq98q)
File: [Finding Aid- Christa Leigh  Steele-Knudslien.pdf](https://github.com/CollegeOfTheHolyCross/dta_sufia/files/1876159/Finding.Aid-.Christa.Leigh.Steele-Knudslien.pdf)

Item: [h702q636h](https://www.digitaltransgenderarchive.net/files/h702q636h)
File: [Joseph L. Nortion Papers.pdf](https://github.com/CollegeOfTheHolyCross/dta_sufia/files/1876169/Joseph.L.Nortion.Papers.pdf)

Item: [f4752g73w](https://www.digitaltransgenderarchive.net/files/f4752g73w)
File: [Captial Region Transgender Community Archive Collection.pdf](https://github.com/CollegeOfTheHolyCross/dta_sufia/files/1876172/Captial.Region.Transgender.Community.Archive.Collection.pdf)

Item: [7w62f822v](https://www.digitaltransgenderarchive.net/files/7w62f822v)
File: [Mariette Pathy Allen Finding Aid.pdf](https://github.com/CollegeOfTheHolyCross/dta_sufia/files/1876173/Mariette.Pathy.Allen.Finding.Aid.pdf)

**Comment by scande3** [comment]
@ntantum -> These are show a file and a thumbnail to me. So the issue is that there is a thumbnail but that thumbnail is not a good representation of the PDF? I'll debug it this weekend as to why their files are creating mostly black thumbnails then but want to confirm as this is different than the original issue.

**Comment by scande3** [comment]
And one additional note: Reuploading the same file to an object should now work. So if a record is created without a a file for some reason, the "edit" should now longer give that error you were experiencing. The root cause is to have some additional debugging information added to it this weekend and do let me know when it still occurs. Thanks.

**Comment by ntantum** [comment]
Sorry for the late response to this, I was out of the office for a couple of days.

But yes, this issue has evolved a little so that now the only problem is that the thumbnail is not a good representation of the PDF (half black, most of the time). I tried re-uploading [this item](https://www.digitaltransgenderarchive.net/files/7w62f822v) and I did not get an error (as your last note said, yay!) but it looks like the thumbnail is still half black. I double checked this on incognito but let me know if you are seeing something else!
